language/it/acp

```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+*
+* @package phpBB Extension - Wiki
+ * @copyright (c) 2015 tas2580 (https://tas2580.net)
+* @license http://opensource.org/licenses/gpl-2.0.php GNU General Public License v2
+*
+*/
+
+/**
+* DO NOT CHANGE
+*/
+if (!defined('IN_PHPBB'))
+{
+	exit;
+}
+if (empty($lang) || !is_array($lang))
+{
+	$lang = array();
+}
+// DEVELOPERS PLEASE NOTE
+//
+// All language files should use UTF-8 as their encoding and the files must not contain a BOM.
+//
+// Placeholders can now contain order information, e.g. instead of
+// 'Page %s of %s' you can (and should) write 'Page %1$s of %2$s', this allows
+// translators to re-order the output of data while ensuring it remains correct
+//
+// You do not need this where single placeholders are used, e.g. 'Message %d' is fine
+// equally where a string contains only two placeholders which are used to wrap text
+// in a url you again do not need to specify an order e.g., 'Click %sHERE%s' is fine
+//
+// Some characters you may want to copy&paste:
+// ’ » “ ” …
+//
+$lang = array_merge($lang, array(
+	'ACP_USERMAP_TITLE'				=> 'Mappa Utente',
+	'ACP_MAP_SETTINGS'				=> 'Impostazioni Mappa',
+	'ACP_USERMAP_ZOOM'				=> 'Zoom',
+	'ACP_USERMAP_ZOOM_EXPLAIN'			=> 'Zoom standard quando si chiama la mappa.',
+	'ACP_COORDINATES'				=> 'Coordinate standard',
+	'ACP_COORDINATES_EXPLAIN'			=> 'Coordinate sulla mappa quando si chiama la sede.',
+	'ACP_GET_COORDINATES'				=> 'Aggiungi mie coordinate',
+	'ACP_SUBMIT'					=> 'Salva impostazioni',
+	'ACP_SAVED'					=> 'Le impostazioni sono state salvate con successo',
+	'ACP_USERMAP_MARKER'				=> 'Indicatore Mappa Utenti',
+	'ACP_USERMAP_MARKER_DESCRIPTION'		=> 'Scegli un Indicatore per gli utenti del gruppo nella mappa utente.',
+	'ACP_USERMAP_COORDINATES'			=> 'Coordinate per la mappa utente',
+	'ACP_USERMAP_COORDINATES_EXPLAIN'		=> 'Le coordinate degli utenti per la mappa utente.',
+	'ACP_USERMAP_LON'				=> 'Gradi di longitudine',
+	'ACP_USERMAP_LAT'				=> 'Gradi di latitudine',
+	'ACP_MAP_TYPE'					=> 'Mappa',
+	'ACP_MAP_TYPE_EXPLAIN'				=> 'Scegli il tipo di  mappa da utilizzare.',
+	'ACP_GOOGLE_API_KEY'				=> 'Google Maps API Key',
+	'ACP_GOOGLE_API_KEY_EXPLAIN'			=> 'Aggiungi la tua <a href="https://developers.google.com/maps/documentation/javascript/get-api-key">API Key</a> per Google Maps.',
+	'ACP_MAP_OSM'					=> 'Apri Mappa',
+	'ACP_MAP_GOOGLE'				=> 'Google Maps',
+	'ACP_KM'					=> 'Chilometri',
+	'ACP_SEARCH_DISTANCE'				=> 'Cerca distanza',
+	'ACP_SEARCH_DISTANCE_EXPLAIN'			=> 'Raggio in cui effettuare la ricerca da parte dell\'utente, salvo diversa indicazione.',
```

Comment:
traduzione controller

<?php
/**
*
* @package phpBB Extension - tas2580 Mobile Notifier
* @copyright (c) 2015 tas2580 (https://tas2580.net)
* @license http://opensource.org/licenses/gpl-2.0.php GNU General Public License v2
*
*/

/**
* DO NOT CHANGE
*/
if (!defined('IN_PHPBB'))
{
	exit;
}

if (empty($lang) || !is_array($lang))
{
	$lang = array();
}

$lang = array_merge($lang, array(
	'USERMAP_TITLE'						=> 'Mappa Utente',
	'BACK_TO_USERMAP'					=> 'Torna a Mappa Utente',
	'DISTANCE'						=> 'Distanza',
	'KM'							=> 'Chilometri',
	'SET_MY_POSITION'					=> 'Impostare la propria posizione qui',
	'CLICK_TO_SET'						=> 'Cliccare sulla mappa per impostare la vostra posizione.',
	'SET_POSITION'						=> 'Imposta Posizione',
	'CONFIRM_COORDINATES_SET'				=> 'Sei sicuro di voler impostare la propria posizione alle seguenti coordinate:<br>Longitudine %1$s<br> Latitudine %2$s',
	'LON'							=> 'Latitudine ',
	'LAT'							=> 'Longitudine',
	'MENU_CLOSE'						=> 'Chiudi',
	'MENU_SEARCH'						=> 'vicinanze Users (%s KM)',
	'SEARCH_EXPLAIN'					=> 'Utenti entro %1$s circa chilometro %2$s, %3$s',
	'USERMAP_SEARCH'					=> 'Cerca Mappa Utente',
	'MEMBERS'						=> 'Utenti',
));

---------------------------------------------------------------------------------------------------------------

info_ap_username

<?php
/**
*
* @package phpBB Extension - tas2580 Mobile Notifier
* @copyright (c) 2015 tas2580 (https://tas2580.net)
* @license http://opensource.org/licenses/gpl-2.0.php GNU General Public License v2
*
*/

/**
* DO NOT CHANGE
*/
if (!defined('IN_PHPBB'))
{
	exit;
}
if (empty($lang) || !is_array($lang))
{
	$lang = array();
}
// DEVELOPERS PLEASE NOTE
//
// All language files should use UTF-8 as their encoding and the files must not contain a BOM.
//
// Placeholders can now contain order information, e.g. instead of
// 'Page %s of %s' you can (and should) write 'Page %1$s of %2$s', this allows
// translators to re-order the output of data while ensuring it remains correct
//
// You do not need this where single placeholders are used, e.g. 'Message %d' is fine
// equally where a string contains only two placeholders which are used to wrap text
// in a url you again do not need to specify an order e.g., 'Click %sHERE%s' is fine
//
// Some characters you may want to copy&paste:
// â€™ Â» â€œ â€ â€¦
//
$lang = array_merge($lang, array(
	'ACP_USERMAP_TITLE'	=> 'Mappa Utente',
));

--------------------------------------------------------------------------------------------------

link

<?php
/**
*
* @package phpBB Extension - Wiki
 * @copyright (c) 2015 tas2580 (https://tas2580.net)
* @license http://opensource.org/licenses/gpl-2.0.php GNU General Public License v2
*
*/

/**
* DO NOT CHANGE
*/
if (!defined('IN_PHPBB'))
{
	exit;
}

if (empty($lang) || !is_array($lang))
{
	$lang = array();
}
// DEVELOPERS PLEASE NOTE
//
// All language files should use UTF-8 as their encoding and the files must not contain a BOM.
//
// Placeholders can now contain order information, e.g. instead of
// 'Page %s of %s' you can (and should) write 'Page %1$s of %2$s', this allows
// translators to re-order the output of data while ensuring it remains correct
//
// You do not need this where single placeholders are used, e.g. 'Message %d' is fine
// equally where a string contains only two placeholders which are used to wrap text
// in a url you again do not need to specify an order e.g., 'Click %sHERE%s' is fine
//
// Some characters you may want to copy&paste:
// â€™ Â» â€œ â€ â€¦
//
$lang = array_merge($lang, array(
	'USERMAP'					=> 'Mappa Utente',

));

---------------------------------------------------------------------------------------------------------------------

permission_usermap

<?php
/**
*
* @package phpBB Extension - Wiki
 * @copyright (c) 2015 tas2580 (https://tas2580.net)
* @license http://opensource.org/licenses/gpl-2.0.php GNU General Public License v2
*
*/

/**
* DO NOT CHANGE
*/
if (!defined('IN_PHPBB'))
{
	exit;
}
if (empty($lang) || !is_array($lang))
{
	$lang = array();
}
// DEVELOPERS PLEASE NOTE
//
// All language files should use UTF-8 as their encoding and the files must not contain a BOM.
//
// Placeholders can now contain order information, e.g. instead of
// 'Page %s of %s' you can (and should) write 'Page %1$s of %2$s', this allows
// translators to re-order the output of data while ensuring it remains correct
//
// You do not need this where single placeholders are used, e.g. 'Message %d' is fine
// equally where a string contains only two placeholders which are used to wrap text
// in a url you again do not need to specify an order e.g., 'Click %sHERE%s' is fine
//
// Some characters you may want to copy&paste:
// â€™ Â» â€œ â€ â€¦
//
$lang = array_merge($lang, array(
	'ACL_U_USERMAP_VIEW'		=> 'Può visualizzare la mappa utente',
	'ACL_U_USERMAP_ADD'		=> 'Può aggiungere mappa utente',
	'ACL_U_USERMAP_SEARCH'	=> 'Può cercare la mappa utente',
));

-----------------------------------------------------------------------------------------------------------

ucp

<?php
/**
*
* @package phpBB Extension - Wiki
 * @copyright (c) 2015 tas2580 (https://tas2580.net)
* @license http://opensource.org/licenses/gpl-2.0.php GNU General Public License v2
*
*/

/**
* DO NOT CHANGE
*/
if (!defined('IN_PHPBB'))
{
	exit;
}

if (empty($lang) || !is_array($lang))
{
	$lang = array();
}
// DEVELOPERS PLEASE NOTE
//
// All language files should use UTF-8 as their encoding and the files must not contain a BOM.
//
// Placeholders can now contain order information, e.g. instead of
// 'Page %s of %s' you can (and should) write 'Page %1$s of %2$s', this allows
// translators to re-order the output of data while ensuring it remains correct
//
// You do not need this where single placeholders are used, e.g. 'Message %d' is fine
// equally where a string contains only two placeholders which are used to wrap text
// in a url you again do not need to specify an order e.g., 'Click %sHERE%s' is fine
//
// Some characters you may want to copy&paste:
// â€™ Â» â€œ â€ â€¦
//
$lang = array_merge($lang, array(
	'UCP_USERMAP_COORDINATES'			=> 'Coordinate per mappa utente',
	'UCP_USERMAP_COORDINATES_EXPLAIN'		=> 'Immettere le coordinate per la <a href="%s">Mappa Utente</a>.',
	'UCP_USERMAP_LON'				=> 'Longitudine',
	'UCP_USERMAP_LAT'				=> 'Latitudine',
	'UCP_USERMAP_GET_COORDINATES'		=> 'Aggiungi le mie coordinate',
));